Library Catalog (COBISS).js

```diff
@@ -0,0 +1,521 @@
+	}
+	const translator = Zotero.loadTranslator('import');
+	translator.setTranslator('32d59d2d-b65a-4da4-b0a3-bdd3cfb979e7'); // RIS
+	translator.setString(fixedRisText);
+	translator.setHandler('itemDone', (_obj, item) => {
+		if (pdfLink) {
+			item.attachments.push({
+				url: pdfLink,
+				title: 'Full Text PDF',
+				mimeType: 'application/pdf'
+			});
+		}
+		else {
+			item.attachments.push({
+				title: 'Snapshot',
+				document: doc
+			});
+		}
+		item.url = url;
+		item.complete();
+	});
+	await translator.translate();
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://plus.cobiss.net/cobiss/si/sl/bib/92020483",
+		"items": [
+			{
+				"itemType": "videoRecording",
+				"title": "Nauk o barvah po Goetheju. DVD 2/3, Poglobitev vsebine nauka o barvah, še posebej poglavja \"Fizične barve\" s prikazom eksperimentov",
+				"creators": [
+					{
+						"lastName": "Kühl",
+						"firstName": "Johannes",
+						"creatorType": "director"
+					}
+				],
+				"date": "2022",
+				"ISBN": "9789619527542",
+				"libraryCatalog": "Library Catalog (COBISS)",
+				"place": "Hvaletinci",
+				"studio": "NID Sapientia",
+				"url": "https://plus.cobiss.net/cobiss/si/sl/bib/92020483",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>Dialogi v slov. in nem. s konsekutivnim prevodom v slov.</p>"
+					},
+					{
+						"note": "<p>Tisk po naročilu</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://plus.cobiss.net/cobiss/si/sl/bib/search?q=*&db=cobib&mat=allmaterials&cof=0_105b-p&pdfrom=01.01.2023",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://plus.cobiss.net/cobiss/si/sl/bib/115256576",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Angel z zahodnega okna",
+				"creators": [
+					{
+						"lastName": "Meyrink",
+						"firstName": "Gustav",
+						"creatorType": "author"
+					}
+				],
+				"date": "2001",
+				"ISBN": "9789616400107",
+				"libraryCatalog": "Library Catalog (COBISS)",
+				"numPages": "2 zv. (216; 203 )",
+				"place": "Ljubljana",
+				"publisher": "Založniški atelje Blodnjak",
+				"series": "Zbirka Blodnjak",
+				"url": "https://plus.cobiss.net/cobiss/si/sl/bib/115256576",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>Prevod dela: Der Engel vom westlichen Fenster</p>"
+					},
+					{
+						"note": "<p>Gustav Meyrink / Herman Hesse: str. 198-200</p>"
+					},
+					{
+						"note": "<p>Magični stekleni vrtovi judovske kulture / Jorge Luis Borges: str. 201-203</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://plus.cobiss.net/cobiss/si/sl/bib/139084803",
+		"items": [
+			{
+				"itemType": "webpage",
```

Comment:
I agree, it's definitely not a webpage... the item is cataloged as TY - ELEC in the RIS, so the RIS import translator identifies it as a webpage according to its exportTypeMap. In fact, it's a report, so probably should have been cataloged as RPRT.

Its correct itemType can be scraped from the text of the catalog page: "Vrsta gradiva - raziskovalno poročilo" ("Type of material - research report"). However, this gets tricky because COBISS is used in 8 countries with 9 different languages, so I could detect which COBISS catalog we're in based on URL (e.g. https://plus.cobiss.net/cobiss/sr/sr/bib/search vs. https://plus.cobiss.net/cobiss/si/sl/bib/search), then have a different item type map for each language. I'd also have to map all of the item types in each COBISS catalog to Zotero item types - in the Serbian COBISS, for example, there are over 50 item types if you look at the 
"врста садржаја" (
type of content) facet on the left: https://plus.cobiss.net/cobiss/sr/sr/bib/search?q=*&db=cobib&mat=allmaterials

So I guess my question is, how far do we go to correct cataloging errors like this vs. using what's provided in the RIS? What I've described would be totally doable, but time-consuming.